### PR TITLE
pd-ctl: add subcommand for region topkeys

### LINKED ISSUE #5007: [Testability Requirements] Supports querying topkeys of region and the number of regions based on a given range of region kyes

### PR DESCRIPTION
**Pull request by bufferflies**
Signed-off-by: bufferflies <1045931706@qq.com>

### What problem does this PR solve?

Issue Number: Close  #5007

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

Code changes

Side effects

Related changes

### Release note

```release-note
 None.
```

### CODE REVIEW

File: server/api/router.go [hunk 1]
```diff
@@ -237,6 +237,7 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	registerFunc(clusterRouter, "/regions/confver", regionsHandler.GetTopConfVerRegions, setMethods("GET"))
 	registerFunc(clusterRouter, "/regions/version", regionsHandler.GetTopVersionRegions, setMethods("GET"))
 	registerFunc(clusterRouter, "/regions/size", regionsHandler.GetTopSizeRegions, setMethods("GET"))
+	registerFunc(clusterRouter, "/regions/topkeys", regionsHandler.GetTopKeysRegions, setMethods("GET"))
```
Comment (rleungx) [inline server/api/router.go:240]: How about
```suggestion
	registerFunc(clusterRouter, "/regions/keys", regionsHandler.GetTopKeysRegions, setMethods("GET"))
```

File: tests/pdctl/region/region_test.go [hunk 2]
```diff
@@ -167,6 +178,7 @@ func (s *regionTestSuite) TestRegion(c *C) {
 	}
 
 	for _, testCase := range testRegionsCases {
+		fmt.Println("case:", testCase.args)
```
Comment (rleungx) [inline tests/pdctl/region/region_test.go:181]: remove it

File: tests/pdctl/region/region_test.go [hunk 3]
```diff
@@ -134,6 +141,10 @@ func (s *regionTestSuite) TestRegion(c *C) {
 		{[]string{"region", "topsize", "2"}, api.TopNRegions(leaderServer.GetRegions(), func(a, b *core.RegionInfo) bool {
 			return a.GetApproximateSize() < b.GetApproximateSize()
 		}, 2)},
+		// region topkeys []limit command
```
Comment (rleungx) [inline tests/pdctl/region/region_test.go:144]:
```suggestion
		// region topkeys [limit] command
```

File: server/api/router.go (hunk 1 above)
Comment (bufferflies) [reply server/api/router.go:240]: the path of `regions/keys` has been used for key scan.

File: tests/pdctl/region/region_test.go (hunk 3 above)
Comment (bufferflies) [reply tests/pdctl/region/region_test.go:144]: fix

File: tests/pdctl/region/region_test.go (hunk 2 above)
Comment (bufferflies) [reply tests/pdctl/region/region_test.go:181]: fix

File: server/api/router.go (hunk 1 above)
Comment (rleungx) [reply server/api/router.go:240]: key scan uses `regions/key`

Comment (bufferflies) [reply server/api/router.go]: fix

File: server/statistics/store_collection.go [hunk 4]
```diff
@@ -193,6 +193,8 @@ func (s *storeStatistics) Collect() {
 	configs["max-merge-region-keys"] = float64(s.opt.GetMaxMergeRegionKeys())
 	configs["region-max-size"] = float64(s.storeConfig.GetRegionMaxSize())
 	configs["region-split-size"] = float64(s.storeConfig.GetRegionSplitSize())
+	configs["region-max-keys"] = float64(s.storeConfig.GetRegionSplitKeys())
+	configs["region-split-keys"] = float64(s.storeConfig.GetRegionMaxKeys())
```
Comment (lhy1024) [inline server/statistics/store_collection.go:197]:
```suggestion
	configs["region-split-keys"] = float64(s.storeConfig.GetRegionSplitKeys())
	configs["region-max-keys"] = float64(s.storeConfig.GetRegionMaxKeys())
```

Comment (bufferflies) [reply server/statistics/store_collection.go]: fix

**Review by lhy1024 — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**